### PR TITLE
Update the dependencies.

### PR DESCRIPTION
**Pull request by Natim**
I have updated some of the dependencies, the code looks better also npm run deploy doesn't work anymore.

We will need to investigate this.

### CODE REVIEW

**Comment by Natim** [comment]
Now it looks better, also we need an update from react-bootstrap apparently.

**Comment by almet** [comment]
This cannot be merged automatically, can you have a look?

File: .eslintrc [hunk 1]
```diff
@@ -0,0 +1,96 @@
+# http://eslint.org/docs/rules/
```
Comment (almet) [inline .eslintrc:1]: Thanks!

**Comment by almet** [comment]
I've updated the branch, the problems were in the package.json (conflict)

**Comment by almet** [comment]
Also I pushed a change that fixes the visibility toggle, master was broken.

**Pull request merged**